.kitchen.yml

```diff
@@ -17,10 +17,8 @@ platforms:
   - name: debian-8
   - name: debian-9
   - name: fedora-28
-  - name: freebsd-10
   - name: freebsd-11
```

Comment:
Nope nope please 😂

Reply:
blame @tas50 he had it before? 

It's passing/working now, do we want to support it till it's too much pain, or drop support today?